whats-new-panel.yaml

```diff
@@ -260,3 +98,63 @@
   template: toolbar_badge
   trigger:
     id: toolbarBadgeUpdate
+- id: WHATS_NEW_PIP_76
+  template: whatsnew_panel_message
+  order: 3
+  content:
+    bucket_id: WHATS_NEW_PIP_76
+    published_date: 1586859903200
+    title:
+      string_id: cfr-whatsnew-pip-fullscreen-header
+    icon_url: data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzIiIGhlaWdodD0iMzIiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PGRlZnM+PGxpbmVhckdyYWRpZW50IHgxPSIwJSIgeTE9IjAlIiB4Mj0iMTAwJSIgeTI9IjEwMCUiIGlkPSJhIj48c3RvcCBzdG9wLWNvbG9yPSIjMEE4NEZGIiBvZmZzZXQ9IjAlIi8+PHN0b3Agc3RvcC1jb2xvcj0iIzAwNjBERiIgb2Zmc2V0PSI0Ny40OTglIi8+PHN0b3Agc3RvcC1jb2xvcj0iIzAwM0VBQSIgb2Zmc2V0PSIxMDAlIi8+PC9saW5lYXJHcmFkaWVudD48L2RlZnM+PGcgZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIj48cmVjdCBmaWxsPSJ1cmwoI2EpIiBmaWxsLXJ1bGU9Im5vbnplcm8iIHg9Ii4wMTEiIHk9Ii4wMDciIHdpZHRoPSIzMiIgaGVpZ2h0PSIzMiIgcng9IjQiLz48cGF0aCBkPSJNMjEgOWMxLjY1NCAwIDMgMS4zNDYgMyAzdjQuMDIxYTEgMSAwIDAgMS0yIDBWMTJjMC0uNTUyLS40NDktMS0xLTFIMTFjLS41NTEgMC0xIC40NDgtMSAxdjhjMCAuNTUyLjQ0OSAxIDEgMWg2YTEgMSAwIDEgMSAwIDJoLTZjLTEuNjU0IDAtMy0xLjM0Ni0zLTN2LThjMC0xLjY1NCAxLjM0Ni0zIDMtM3ptMS41IDEwYS41LjUgMCAwIDEgMCAxaC0uNzkzbDIuMTQ2IDIuMTQ2YS41LjUgMCAxIDEtLjcwNy43MDhMMjEgMjAuNzA3di43OTNhLjUuNSAwIDAgMS0xIDB2LTJhLjQ5OC40OTggMCAwIDEgLjUtLjV6TTE3IDEyYTEgMSAwIDAgMSAxIDF2M2ExIDEgMCAwIDEtMSAxaC01YTEgMSAwIDAgMS0xLTF2LTNhMSAxIDAgMCAxIDEtMXoiIGZpbGw9IiNGRkYiLz48L2c+PC9zdmc+
+    icon_alt:
+      string_id: cfr-whatsnew-pip-fullscreen-icon-alt
+    body:
+      string_id: cfr-whatsnew-pip-fullscreen-body
+    cta_url: https://support.mozilla.org/1/firefox/%VERSION%/%OS%/%LOCALE%/about-p2p
+    cta_type: OPEN_URL
+    link_text:
+      string_id: cfr-whatsnew-pip-cta
+  targeting: firefoxVersion >= 76
+  trigger:
+    id: whatsNewPanelOpened
+- id: WHATS_NEW_VULNERABLE_PASSWORDS
+  template: whatsnew_panel_message
+  order: 2
+  content:
+    bucket_id: WHATS_NEW_VULNERABLE_PASSWORDS
+    published_date: 1586859903200
```

Comment:
Shall we just set this to the release date timestamp (1588636800000 i.e. 5th May 2020) for 76?